components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -246,6 +246,16 @@ sealed class ContentAction : BrowserAction() {
      * Removes the [SearchRequest] of the [ContentState] with the given [sessionId].
      */
     data class ConsumeSearchRequestAction(val sessionId: String) : ContentAction()
+
+    /**
+     * Updates the [fullScreenEnabled] with the given [sessionId].
+     */
+    data class FullScreenChangedAction(val sessionId: String, val fullScreenEnabled: Boolean) : ContentAction()
+
+    /**
+     * Updates the [layoutInDisplayCutoutMode] with the given [sessionId].
+     */
+    data class ViewportFitChangedAction(val sessionId: String, val layoutInDisplayCutoutMode: Int) : ContentAction()
```

Comment:
Same here. Docs for the int value or point to where one place in the codebase to find them.

Reply:
Agreed